### ISSUE TITLE
Inviting some via private folder

### ISSUE THREAD

**Issue by imnewkidonblock** [keybase/keybase-issues#2877]
Is a twitter handle the only way?

The docs say I can create one that's private/username,friend@twitter

Is there a way to also set one up private/username,friend@emailservice.com?

**Comment by malgorithms** [comment]
no, not by email, but you could just invite them to join and ask their keybase username. then it's `private/yourname,theirname` .

**Comment by imnewkidonblock** [comment]
Sure, that's what I've done. I guess since we have limited invites, when there is a twitter handle we sort of extend that short list.

**Comment by imnewkidonblock** [comment]
OK, follow u p Q. If I want to share a folder with user friend, I would ideally call the folder 
\private\username ,friend

But Windows doesn't like that format. How do I do that with Windows? The client doesn't seem allow it--I have to go to Win file manager.

**Comment by imnewkidonblock** [comment]
I figured it out. Perhaps an update to the client or documentation to be clear that to share files with someone, at least privately, you need to click on "chat." I would suggest changing the nomenclature to be more clear.
